astropy/cosmology/units.py

```diff
@@ -150,23 +238,52 @@ def with_redshift(cosmology=None, *,
     ----------
     cosmology : `~astropy.cosmology.Cosmology`, str, or None, optional
         A cosmology realization or built-in cosmology's name (e.g. 'Planck18').
-        If None, will use the default cosmology
+        If `None`, will use the default cosmology
         (controlled by :class:`~astropy.cosmology.default_cosmology`).
 
-    Tcmb : bool (optional, keyword-only)
-        Whether to create a CMB temperature <-> redshift equivalency, using
-        ``Cosmology.Tcmb``. Default is `True`.
+    distance : {'comoving', 'lookback', 'luminosity'} or None (optional, keyword-only)
+        The distance type or `None`. Default is 'comoving'.
     hubble : bool (optional, keyword-only)
         Whether to create a Hubble parameter <-> redshift equivalency, using
         ``Cosmology.H``. Default is `True`.
+    Tcmb : bool (optional, keyword-only)
+        Whether to create a CMB temperature <-> redshift equivalency, using
+        ``Cosmology.Tcmb``. Default is `True`.
 
     atzkw : dict or None (optional, keyword-only)
         keyword arguments for :func:`~astropy.cosmology.z_at_value`
 
     Returns
     -------
     `~astropy.units.equivalencies.Equivalency`
-        With equivalencies between redshift and temperature / Hubble.
+        With equivalencies between redshift and temperature / distance / Hubble.
```

Comment:
Keep order the same, i.e., "distance / Hubble / temperature" (or swap in arguments)

Reply:
Yeah, I'm just trying to keep it alphabetized.